client/src/components/Interest.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+class Interests extends React.Component {
+      render() {
+            return ( 
```

Comment:
With a component this small, we could easily convert it to a functional component. Especially because there is no interacting with state on here.

This also goes for a few of your other components.